### ISSUE TITLE
C2 phase failed due to No cached parameters found

### ISSUE THREAD

**Issue by maginaro** [filecoin-project/lotus#5153]
Please provide all the information requested here to help us troubleshoot "commit failed" issues.
If the information requested is missing, we will probably have to just ask you to provide it anyway,
before we can help debug.

**Describe the problem**

Sealing Commit2 phase failed due to proof-parameters , but on my worker machine proof-parameters exits and worker start normally.

**Sectors status**

SectorID:	27
Status:		PreCommit1
CIDcommD:	baga6ea4seaqao7s73y24kcutaosvacpdjgfe5pw76ooefnyqw4ynr3d2y6x2mpq
CIDcommR:	bagboea4b5abcavmocqgcxcf6mfl3d4iw62egkkxck36qgdjigeysliubhqre73sh
Ticket:		0bfc6c66f359c26e0c5a15681f2ac5637e0eaddc9a00d8422569a6fac904fe89
TicketH:	302506
Seed:		05d05daa11d34866942d77f047954984abc99b481c122c3c238f8de503122ab2
SeedH:		305485
Precommit:	bafy2bzacedsbrvxuagciqqj5cshdtx7fsj6loswjetce5utsz7xuktp3eseay
Commit:		<nil>
Proof:
Deals:		[0]
Retries:	0

Event Log:
0.	2020-12-08 14:23:06 +0800 CST:	[event;sealing.SectorStartCC]	{"User":{"ID":27,"SectorType":8,"Pieces":[{"Piece":{"Size":34359738368,"PieceCID":{"/":"baga6ea4seaqao7s73y24kcutaosvacpdjgfe5pw76ooefnyqw4ynr3d2y6x2mpq"}},"DealInfo":null}]}}
1.	2020-12-08 14:23:06 +0800 CST:	[event;sealing.SectorPacked]	{"User":{"FillerPieces":null}}
2.	2020-12-08 14:23:06 +0800 CST:	[event;sealing.SectorTicket]	{"User":{"TicketValue":"C/xsZvNZwm4MWhVoHyrFY34OrdyaANhCJWmm+skE/ok=","TicketEpoch":302506}}
3.	2020-12-08 17:53:14 +0800 CST:	[event;sealing.SectorRestart]	{"User":{}}
4.	2020-12-08 19:27:18 +0800 CST:	[event;sealing.SectorRestart]	{"User":{}}
5.	2020-12-08 19:30:10 +0800 CST:	[event;sealing.SectorRestart]	{"User":{}}
6.	2020-12-08 22:43:00 +0800 CST:	[event;sealing.SectorSealPreCommit1Failed]	{"User":{}}
	seal pre commit(1) failed: storage call error 0: task aborted
7.	2020-12-08 22:44:00 +0800 CST:	[event;sealing.SectorRetrySealPreCommit1]	{"User":{}}
8.	2020-12-08 22:44:00 +0800 CST:	[event;sealing.SectorSealPreCommit1Failed]	{"User":{}}
	seal pre commit(1) failed: storage call error 0: failed to acquire sector {93554 27} from remote(1): sector not found
9.	2020-12-08 22:45:00 +0800 CST:	[event;sealing.SectorRetrySealPreCommit1]	{"User":{}}
10.	2020-12-09 03:54:31 +0800 CST:	[event;sealing.SectorPreCommit1]	{"User":{"PreCommit1Out":"eyJyZWdpc3RlcmVkX3Byb29mIjoiU3RhY2tlZERyZzMyR2lCVjFfMSIsImxhYmVscyI6eyJTdGFja2VkRHJnMzJHaUJWMSI6eyJsYWJlbHMiOlt7InBhdGgiOiIvc3RvcmFnZS1ob3QvbG90dXMtd29ya2VyL0pYVC1TVzE2LTcyMTMtQU4wMVROOUwtNC04Ny1QMS0xL2NhY2hlL3MtdDA5MzU1NC0yNyIsImlkIjoibGF5ZXItMSIsInNpemUiOjEwNzM3NDE4MjQsInJvd3NfdG9fZGlzY2FyZCI6N30seyJwYXRoIjoiL3N0b3JhZ2UtaG90L2xvdHVzLXdvcmtlci9KWFQtU1cxNi03MjEzLUFOMDFUTjlMLTQtODctUDEtMS9jYWNoZS9zLXQwOTM1NTQtMjciLCJpZCI6ImxheWVyLTIiLCJzaXplIjoxMDczNzQxODI0LCJyb3dzX3RvX2Rpc2NhcmQiOjd9LHsicGF0aCI6Ii9zdG9yYWdlLWhvdC9sb3R1cy13b3JrZXIvSlhULVNXMTYtNzIxMy1BTjAxVE45TC00LTg3LVAxLTEvY2FjaGUvcy10MDkzNTU0LTI3IiwiaWQiOiJsYXllci0zIiwic2l6ZSI6MTA3Mzc0MTgyNCwicm93c190b19kaXNjYXJkIjo3fSx7InBhdGgiOiIvc3RvcmFnZS1ob3QvbG90dXMtd29ya2VyL0pYVC1TVzE2LTcyMTMtQU4wMVROOUwtNC04Ny1QMS0xL2NhY2hlL3MtdDA5MzU1NC0yNyIsImlkIjoibGF5ZXItNCIsInNpemUiOjEwNzM3NDE4MjQsInJvd3NfdG9fZGlzY2FyZCI6N30seyJwYXRoIjoiL3N0b3JhZ2UtaG90L2xvdHVzLXdvcmtlci9KWFQtU1cxNi03MjEzLUFOMDFUTjlMLTQtODctUDEtMS9jYWNoZS9zLXQwOTM1NTQtMjciLCJpZCI6ImxheWVyLTUiLCJzaXplIjoxMDczNzQxODI0LCJyb3dzX3RvX2Rpc2NhcmQiOjd9LHsicGF0aCI6Ii9zdG9yYWdlLWhvdC9sb3R1cy13b3JrZXIvSlhULVNXMTYtNzIxMy1BTjAxVE45TC00LTg3LVAxLTEvY2FjaGUvcy10MDkzNTU0LTI3IiwiaWQiOiJsYXllci02Iiwic2l6ZSI6MTA3Mzc0MTgyNCwicm93c190b19kaXNjYXJkIjo3fSx7InBhdGgiOiIvc3RvcmFnZS1ob3QvbG90dXMtd29ya2VyL0pYVC1TVzE2LTcyMTMtQU4wMVROOUwtNC04Ny1QMS0xL2NhY2hlL3MtdDA5MzU1NC0yNyIsImlkIjoibGF5ZXItNyIsInNpemUiOjEwNzM3NDE4MjQsInJvd3NfdG9fZGlzY2FyZCI6N30seyJwYXRoIjoiL3N0b3JhZ2UtaG90L2xvdHVzLXdvcmtlci9KWFQtU1cxNi03MjEzLUFOMDFUTjlMLTQtODctUDEtMS9jYWNoZS9zLXQwOTM1NTQtMjciLCJpZCI6ImxheWVyLTgiLCJzaXplIjoxMDczNzQxODI0LCJyb3dzX3RvX2Rpc2NhcmQiOjd9LHsicGF0aCI6Ii9zdG9yYWdlLWhvdC9sb3R1cy13b3JrZXIvSlhULVNXMTYtNzIxMy1BTjAxVE45TC00LTg3LVAxLTEvY2FjaGUvcy10MDkzNTU0LTI3IiwiaWQiOiJsYXllci05Iiwic2l6ZSI6MTA3Mzc0MTgyNCwicm93c190b19kaXNjYXJkIjo3fSx7InBhdGgiOiIvc3RvcmFnZS1ob3QvbG90dXMtd29ya2VyL0pYVC1TVzE2LTcyMTMtQU4wMVROOUwtNC04Ny1QMS0xL2NhY2hlL3MtdDA5MzU1NC0yNyIsImlkIjoibGF5ZXItMTAiLCJzaXplIjoxMDczNzQxODI0LCJyb3dzX3RvX2Rpc2NhcmQiOjd9LHsicGF0aCI6Ii9zdG9yYWdlLWhvdC9sb3R1cy13b3JrZXIvSlhULVNXMTYtNzIxMy1BTjAxVE45TC00LTg3LVAxLTEvY2FjaGUvcy10MDkzNTU0LTI3IiwiaWQiOiJsYXllci0xMSIsInNpemUiOjEwNzM3NDE4MjQsInJvd3NfdG9fZGlzY2FyZCI6N31dLCJfaCI6bnVsbH19LCJjb25maWciOnsicGF0aCI6Ii9zdG9yYWdlLWhvdC9sb3R1cy13b3JrZXIvSlhULVNXMTYtNzIxMy1BTjAxVE45TC00LTg3LVAxLTEvY2FjaGUvcy10MDkzNTU0LTI3IiwiaWQiOiJ0cmVlLWQiLCJzaXplIjoyMTQ3NDgzNjQ3LCJyb3dzX3RvX2Rpc2NhcmQiOjd9LCJjb21tX2QiOls3LDEyNiw5NSwyMjIsNTMsMTk3LDEwLDE0NywzLDE2NSw4MCw5LDIyNyw3MywxMzgsNzgsMTkwLDIyMywyNDMsMTU2LDY2LDE4MywxNiwxODMsNDgsMjE2LDIzNiwxMjIsMTk5LDE3NSwxNjYsNjJdfQ=="}}
11.	2020-12-09 05:05:02 +0800 CST:	[event;sealing.SectorPreCommit2]	{"User":{"Sealed":{"/":"bagboea4b5abcavmocqgcxcf6mfl3d4iw62egkkxck36qgdjigeysliubhqre73sh"},"Unsealed":{"/":"baga6ea4seaqao7s73y24kcutaosvacpdjgfe5pw76ooefnyqw4ynr3d2y6x2mpq"}}}
12.	2020-12-09 05:05:04 +0800 CST:	[event;sealing.SectorPreCommitted]	{"User":{"Message":{"/":"bafy2bzacedsbrvxuagciqqj5cshdtx7fsj6loswjetce5utsz7xuktp3eseay"},"PreCommitDeposit":"105351544302952752","PreCommitInfo":{"SealProof":8,"SectorNumber":27,"SealedCID":{"/":"bagboea4b5abcavmocqgcxcf6mfl3d4iw62egkkxck36qgdjigeysliubhqre73sh"},"SealRandEpoch":302506,"DealIDs":[],"Expiration":1855008,"ReplaceCapacity":false,"ReplaceSectorDeadline":0,"ReplaceSectorPartition":0,"ReplaceSectorNumber":0}}}
13.	2020-12-09 06:30:30 +0800 CST:	[event;sealing.SectorPreCommitLanded]	{"User":{"TipSet":"AXGg5AIg90dcSzICwy8uIwwjwP09Pc9J29tk4M4m6n6cGzrBF0QBcaDkAiDnkBuzkpdkOXQ0UAPy7BXtsBOllCpsEpQBHjYmgZmOSgFxoOQCINGUmXfAV0qLwvvMO0E3cB0P+umfNxrmcFdIjtlrlgjQAXGg5AIgyCgBSnd+u82rUovZn19eCQad41iUH0qd8C28SkCXpNoBcaDkAiC/jwA2YS16kayxQKV84MoKf/PXbAY888MluftPU3HP6gFxoOQCILL4jbs3z3XhoRKcOjVzQDuDJ57tN4d47oNPxjVNHZvSAXGg5AIgY2vUE0+gt8xrEc1Md8qyvVJo2yk0EqfAxoEv5HcfDWQ="}}
14.	2020-12-09 07:45:30 +0800 CST:	[event;sealing.SectorSeedReady]	{"User":{"SeedValue":"BdBdqhHTSGaULXfwR5VJhKvJm0gcEiw8I4+N5QMSKrI=","SeedEpoch":305485}}
15.	2020-12-09 07:45:34 +0800 CST:	[event;sealing.SectorComputeProofFailed]	{"User":{}}
	computing seal proof failed(2): storage call error 0: No cached parameters found for stacked-proof-of-replication-merkletree-poseidon_hasher-8-8-0-sha256_hasher-82a357d2f2ca81dc61bb45f4a762807aedee1b0a53fd6c4e77b46a01bfef7820 [failure finding /storage-hot/filecoin-proof-parameters/v28-stacked-proof-of-replication-merkletree-poseidon_hasher-8-8-0-sha256_hasher-82a357d2f2ca81dc61bb45f4a762807aedee1b0a53fd6c4e77b46a01bfef7820.params]
16.	2020-12-09 07:46:34 +0800 CST:	[event;sealing.SectorRetryComputeProof]	{"User":{}}
17.	2020-12-09 07:46:36 +0800 CST:	[event;sealing.SectorComputeProofFailed]	{"User":{}}
	computing seal proof failed(2): storage call error 0: No cached parameters found for stacked-proof-of-replication-merkletree-poseidon_hasher-8-8-0-sha256_hasher-82a357d2f2ca81dc61bb45f4a762807aedee1b0a53fd6c4e77b46a01bfef7820 [failure finding /storage-hot/filecoin-proof-parameters/v28-stacked-proof-of-replication-merkletree-poseidon_hasher-8-8-0-sha256_hasher-82a357d2f2ca81dc61bb45f4a762807aedee1b0a53fd6c4e77b46a01bfef7820.params]
18.	2020-12-09 07:47:36 +0800 CST:	[event;sealing.SectorRetryComputeProof]	{"User":{}}
19.	2020-12-09 07:47:38 +0800 CST:	[event;sealing.SectorComputeProofFailed]	{"User":{}}
	computing seal proof failed(2): storage call error 0: No cached parameters found for stacked-proof-of-replication-merkletree-poseidon_hasher-8-8-0-sha256_hasher-82a357d2f2ca81dc61bb45f4a762807aedee1b0a53fd6c4e77b46a01bfef7820 [failure finding /storage-hot/filecoin-proof-parameters/v28-stacked-proof-of-replication-merkletree-poseidon_hasher-8-8-0-sha256_hasher-82a357d2f2ca81dc61bb45f4a762807aedee1b0a53fd6c4e77b46a01bfef7820.params]
20.	2020-12-09 07:48:38 +0800 CST:	[event;sealing.SectorSealPreCommit1Failed]	{"User":{}}
	consecutive compute fails
21.	2020-12-09 07:49:38 +0800 CST:	[event;sealing.SectorRetrySealPreCommit1]	{"User":{}}

**Lotus miner logs**

2020-12-09T07:42:30.254+0800	WARN	sectors	storage-sealing/states_sealing.go:356	revert in interactive commit sector step
2020-12-09T07:45:30.361+0800	INFO	sectors	storage-sealing/states_sealing.go:382	scheduling seal proof computation...
2020-12-09T07:45:30.362+0800	INFO	sectors	storage-sealing/states_sealing.go:384	KOMIT 27 0bfc6c66f359c26e0c5a15681f2ac5637e0eaddc9a00d8422569a6fac904fe89(302506); 05d05daa11d34866942d77f047954984abc99b481c122c3c238f8de503122ab2(305485); [{34359738368 baga6ea4seaqao7s73y24kcutaosvacpdjgfe5pw76ooefnyqw4ynr3d2y6x2mpq}]; r:626167626f656134623561626361766d6f63716763786366366d666c3364346977363265676b6b78636b33367167646a69676579736c6975626871726537337368; d:626167613665613473656171616f377337337932346b637574616f7376616370646a67666535707737366f6f65666e79717734796e72336432793678326d7071
2020-12-09T07:45:30.401+0800	DEBUG	advmgr	sector-storage/sched.go:463	SCHED try assign sqi:0 sector 27 to window 2
2020-12-09T07:45:30.401+0800	DEBUG	advmgr	sector-storage/sched.go:470	SCHED ASSIGNED sqi:0 sector 27 task seal/v0/commit/1 to window 2
2020-12-09T07:45:30.401+0800	DEBUG	advmgr	sector-storage/sched_worker.go:367	assign worker sector 27
2020-12-09T07:45:30.401+0800	DEBUG	advmgr	sector-storage/sched.go:449	SCHED Acceptable win: []
2020-12-09T07:45:30.425+0800	DEBUG	advmgr	sector-storage/sched_worker.go:277	task done	{"workerid": [34,8,201,72,182,131,64,58,169,173,126,220,66,72,133,100]}
2020-12-09T07:45:32.841+0800	DEBUG	advmgr	sector-storage/sched.go:449	SCHED Acceptable win: [[7 8]]
2020-12-09T07:45:32.841+0800	DEBUG	advmgr	sector-storage/sched.go:463	SCHED try assign sqi:0 sector 27 to window 7
2020-12-09T07:45:32.841+0800	DEBUG	advmgr	sector-storage/sched.go:470	SCHED ASSIGNED sqi:0 sector 27 task seal/v0/commit/2 to window 7
2020-12-09T07:45:32.841+0800	DEBUG	advmgr	sector-storage/sched_worker.go:367	assign worker sector 27
2020-12-09T07:45:32.842+0800	DEBUG	advmgr	sector-storage/sched.go:449	SCHED Acceptable win: []
2020-12-09T07:45:32.843+0800	DEBUG	advmgr	sector-storage/sched_worker.go:277	task done	{"workerid": [95,178,126,37,35,118,71,167,185,152,160,65,233,17,190,191]}
2020-12-09T07:45:34.054+0800	INFO	sectors	storage-sealing/states_failed.go:26	ComputeProofFailed(27), waiting 59.945407246s before retrying
2020-12-09T07:46:34.013+0800	INFO	sectors	storage-sealing/states_sealing.go:382	scheduling seal proof computation...

**Lotus worker logs**

2020-12-09T07:45:33.869 INFO filcrypto::proofs::api > seal_commit_phase2: start
2020-12-09T07:45:34.007 INFO filecoin_proofs::api::seal > seal_commit_phase2:start: SectorId(27)
2020-12-09T07:45:34.009 INFO filecoin_proofs::caches > trying parameters memory cache for: STACKED[34359738368]
2020-12-09T07:45:34.009 INFO filecoin_proofs::caches > no params in memory cache for STACKED[34359738368]
2020-12-09T07:45:34.010 INFO storage_proofs_core::parameter_cache > parameter set identifier for cache: layered_drgporep::PublicParams{ graph: stacked_graph::StackedGraph{expansion_degree: 8 base_graph: drgraph::BucketGraph{size: 1073741824; degree: 6; hasher: poseidon_hasher} }, challenges: LayerChallenges { layers: 11, max_count: 18 }, tree: merkletree-poseidon_hasher-8-8-0 }
2020-12-09T07:45:34.011 INFO storage_proofs_core::parameter_cache > ensuring that all ancestor directories for: "/storage-hot/filecoin-proof-parameters/v28-stacked-proof-of-replication-merkletree-poseidon_hasher-8-8-0-sha256_hasher-82a357d2f2ca81dc61bb45f4a762807aedee1b0a53fd6c4e77b46a01bfef7820.params" exist
2020-12-09T07:45:34.011 INFO storage_proofs_core::parameter_cache > checking cache_path: "/storage-hot/filecoin-proof-parameters/v28-stacked-proof-of-replication-merkletree-poseidon_hasher-8-8-0-sha256_hasher-82a357d2f2ca81dc61bb45f4a762807aedee1b0a53fd6c4e77b46a01bfef7820.params" for parameters
2020-12-09T07:45:34.014 INFO filcrypto::proofs::api > seal_commit_phase2: finish

**Version**

lotus version 1.2.1+git.df66f48f6

**Comment by Reiers** [comment]
Hi @maginaro 

Thanks for the report. The issue is outdated.

If you are still experiencing problems, open a new ticket or ask for help in our [Discussions forum.](https://github.com/filecoin-project/lotus/discussions)
